pkg/daemon/update.go

```diff
@@ -1908,8 +1908,29 @@ func (dn *Daemon) updateOS(config *mcfgv1.MachineConfig, osImageContentDir strin
 func (dn *Daemon) updateLayeredOS(config *mcfgv1.MachineConfig) error {
 	newURL := config.Spec.OSImageURL
 	glog.Infof("Updating OS to layered image %s", newURL)
-	if err := dn.NodeUpdaterClient.RebaseLayered(newURL); err != nil {
-		return fmt.Errorf("failed to update OS to %s : %w", newURL, err)
+
+	newEnough, err := RpmOstreeIsNewEnoughForLayering()
+	if err != nil {
+		return err
+	}
+	glog.Infof("rpm-ostree is new enough for layering: %v", newEnough)
```

Comment:
I think this should be printed down in the else section where we are calling RebaseLayered()

Reply:
I don't quite understand what you meant here; I think what we more want to highlight is the workaround case right?

Reply:
I moved logging to the workaround case